features/step_definitions/sign_up.rb

```diff
@@ -0,0 +1,80 @@
+require 'faker'
+
+# Navigation to sign up page:
+
+Given('I am on the login page') do
+  navigate_to(ENV['URL'])
+end
+
+And('I click on the registration button') do
+  click(:id, 'sign-up')
+end
+
+# Successful registration:
+
+When('I fill all the required fields') do
+  enter_text(:id, Faker::Name.first_name, 'firstname')
+  enter_text(:id, Faker::Name.last_name, 'lastname')
+  enter_text(:id, Faker::Internet.email(domain: 'wolox.com.ar'), 'email')
+  enter_text(:id, Faker::Internet.password, 'password')
+end
+
+And('I click on the submit button') do
+  click(:id, 'submit_register')
+end
+
+Then('I will be redirected to the login page') do
+  wait(2) # por qué??
+  expect($driver.current_url).to eq(ENV['URL'] + '/login')
+end
+
+# Unsuccessful registration: Empty parameters
+
+When('I leave the {string} field empty') do |field|
+  enter_text(:id, '', field)
+end
+
+And('I click some other field') do
+  click(:id, 'unauth_container')
+end
+
+Then('A required field message will pop up for {string}') do |field|
+  case field
+  when 'firstname' || 'lastname'
+    field = 'first_name_register_span'
+  when 'email' || 'password'
+    field += '_register_span_req'
+  end
+  alert = check_element_presence(:id, field, true)
```

Comment:
the validation is ok, but you can complement it by capturing the error message and comparing it with the correct one in this case   "**This field is required"** which you can send from the feature as a parameter